### PR TITLE
Release notes for 0.8.2

### PR DESCRIPTION
**Pull request by johannes-b**
Signed-off-by: Johannes <johannes.braeuer@dynatrace.com>

### CODE REVIEW

File: releasenotes/releasenotes_develop.md [hunk 1]
```diff
@@ -1 +1,123 @@
...
+  - Keptn API: `configure monitoring` not functioning according to spec [3638](https://github.com/keptn/keptn/issues/3638)
+  - Keptn CLI: Disable Kube context check [3666](https://github.com/keptn/keptn/issues/3666)
+  - Homebrew installed CLI fails install with 'Malformed constraint: ""' [3805](https://github.com/keptn/keptn/issues/3805)
+
+</p>
+</details>
+
+## Upgrade to 0.8.2
+
+- The upgrade from 0.8.x to 0.8.2 is supported by the `keptn upgrade` command. Find the documentation here: [Upgrade from Keptn 0.8.x to 0.8.2](https://keptn.sh/docs/0.8.x/operate/upgrade/#upgrade-from-keptn-0-8-1-to-0-8-2)
```
Comment (ermin-muratovic) [inline releasenotes/releasenotes_develop.md:123]: The title says "Upgrade from Keptn 0.8.x to 0.8.2", but the link is `https://keptn.sh/docs/0.8.x/operate/upgrade/#upgrade-from-keptn-_0-8-1_-to-0-8-2`. Should we make that equal?

**Review by ermin-muratovic — CHANGES_REQUESTED** [review]
lgtm

File: releasenotes/releasenotes_develop.md [hunk 2]
```diff
@@ -58,11 +70,18 @@ Implemented **Keptn spec** version: [0.2.1](https://github.com/keptn/spec/tree/0
 <details><summary>Bridge</summary>
 <p>
 
+- OAuth/OpenID connect based login for Keptn bridge [3448](https://github.com/keptn/keptn/issues/3448)
```
Comment (jsoref) [inline releasenotes/releasenotes_develop.md:73]: Please add `OAuth` to the `expect.txt` file -- something's wrong w/ the bot -- I need to figure out why it flaked out.

**Review by jsoref — CHANGES_REQUESTED** [review]

File: releasenotes/releasenotes_develop.md (hunk 2 above)
Comment (jsoref) [reply releasenotes/releasenotes_develop.md:73]: Oh, you did, sigh. Sorry.

Comment (jsoref) [reply releasenotes/releasenotes_develop.md:73]: Fwiw, the buggy error reporting from the spell checker is already fixed in prerelease (yay). I need to work on writing release notes and releasing -- just like you folks :-)

**Review by ermin-muratovic — APPROVED** [review]
lgtm

File: releasenotes/releasenotes_develop.md [hunk 3]
```diff
@@ -1 +1,123 @@
...
+  - Keptn API: `configure monitoring` not functioning according to spec [3638](https://github.com/keptn/keptn/issues/3638)
+  - Keptn CLI: Disable Kube context check [3666](https://github.com/keptn/keptn/issues/3666)
+  - Homebrew installed CLI fails install with 'Malformed constraint: ""' [3805](https://github.com/keptn/keptn/issues/3805)
+
+</p>
+</details>
+
+## Upgrade to 0.8.2
+
+- The upgrade from 0.8.1 to 0.8.2 is supported by the `keptn upgrade` command. Find the documentation here: [Upgrade from Keptn 0.8.x to 0.8.2](https://keptn.sh/docs/0.8.x/operate/upgrade/#upgrade-from-keptn-0-8-1-to-0-8-2)
```
Comment (jsoref) [inline releasenotes/releasenotes_develop.md:123]: Fwiw, I'd suggest this style (but it probably should be applied to multiple files):

```suggestion
- Upgrading from 0.8.1 to 0.8.2 is supported by the `keptn upgrade` command. For more information, see the documentation: [Upgrade from Keptn 0.8.x to 0.8.2](https://keptn.sh/docs/0.8.x/operate/upgrade/#upgrade-from-keptn-0-8-1-to-0-8-2)
```

**Pull request merged**